Fix issue with SELinux permissions

Andrew5569 opened the pull request:
resolves https://github.com/ItayGarin/ktrl/issues/24

The issue can be resolved by copying the ktrl bin from `~/.cargo/bin/ktrl` to `/usr/local/bin/ktrl`, updating the permissions on the new bin, and lastly updating the bin path in `etc/ktrl.service`.

I tried creating a link but it didn't resolve the issue so I decided to just cp it.

AndrewSmithDev committed:
fix issue with SELinux permissions

ItayGarin commented:
Thanks for the PR Andrew!

I'm wondering whether we should make this more of an optional step.
The downside to copying the executable is that you'd have to remember doing that every time you upgrade ktrl (e.g. via crago)

WDYT?

AndrewSmithDev committed:
change cargo install directory

specify install cargo install directory
instead of copying the ktrl bin to /usr/local/bin/ktrl

ghost commented:
Good point. 

I looked into cargo a bit and found that you can specify where to install the project using the `--root` argument: `sudo cargo install --root /usr/local ktrl`.This should be a much better solution.

ItayGarin commented:
Looks good! Thanks! 👍
